test/unit/process/test_process_set_policy.hpp

```diff
@@ -98,36 +98,36 @@ void testProcessSet(const std::function<ProcessSetPolicy(
   compare_pair(*(++elem), index_pair(4, 0));
   compare_pair(*(++elem), index_pair(4, 2));
 
-  auto builder = SparseMatrixPolicy<double>::create(5).number_of_blocks(2).initial_value(100.0);
+  auto builder = SparseMatrixPolicy<double>::create(5).number_of_blocks(2).initial_value(0.0);
```

Comment:
I would suggest leaving the initial value 100.0 and modifying the expectations to be `100 - ...` instead of `100 + ...`, just because the functions should preserve any existing values in the Jacobian (this may be important when we add aerosol processes that may have separate functions that modify the Jacobian)

Reply:
Good point. But ins this case, we can not negate the matrix like the current implementation. If there is an initial value of Jacobian matrix such as `100`, then the calculated Jacobian will be `100+a-b` and the negative Jacobian will be `-100-a+b`. To compute this negative Jacobian matrix correctly, we will then have to write a separate loop to negate it.

Reply:
I'm not sure that applies to this test - this just tests the function that subtracts jacobian elements from a matrix. It shouldn't care what values are there to start with. If all the matrix elements start at `100`, and jacobian terms being subtracted from a particular element are `a` and `-b`, then the final value for that element should be `100-a+b`

Reply:
Agreed. Thanks for your suggestions.

Reply:
Done.